index.html

```diff
@@ -5,18 +5,19 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <link href="styles.css" rel="stylesheet">
+  <script src="script.js" async></script>
```

Comment:
I moved the script tag to the head and made it async so that it will load in the background instead of having to wait for the page to load before it starts loading.